app/react/portainer/environments/utils/index.ts

```diff
@@ -67,6 +67,13 @@ export function isLocalEnvironment(environment: Environment) {
   );
 }
 
+export function isDockerAPIEnvironment(environment: Environment) {
+  return (
+    environment.URL.startsWith('tcp://') &&
+    isDockerEnvironment(environment.Type)
```

Comment:
`isDockerEnvironment` returns true also for agent. docker api is only `EnvironmentType.Docker`
```suggestion
    environment.Type === EnvironmentType.Docker
```

Reply:
fixed